note.h

```diff
@@ -73,13 +73,13 @@
 // originates from the Notecard, which synchronizes the time from both the cell network and GPS.
 typedef unsigned long int JTIME;
 
-// If we're building the tests, STATIC is defined to nothing. This allows the
-// tests to access the static functions in note-c. Among other things, this
+// If we're building the tests, NOTE_C_STATIC is defined to nothing. This allows
+// the tests to access the static functions in note-c. Among other things, this
 // let's us mock these normally static functions.
-#ifdef TEST
-#define STATIC
+#ifdef NOTE_C_TEST
+#define NOTE_C_STATIC
 #else
-#define STATIC static
+#define NOTE_C_STATIC static
```

Comment:
Can this be moved into `n_helpers.c`, or into a test only header file that lives in `test/include`?

Reply:
> Can this be moved into n_helpers.c

I don't love the idea of moving it into n_helpers.c because other source files might have functions we want to mock at some point.

> or into a test only header file that lives in test/include

And then include that header in n_helpers.c (and elsewhere, if needed)? That could work. Please confirm my understanding, though.